tests/testAddToCoadd.py

```diff
@@ -44,19 +44,16 @@
     display
 except NameError:
     display = False
-    Verbosity = 0 # increase to see trace
+    Verbosity = 0  # increase to see trace
 
 pexLog.Trace_setVerbosity("lsst.coadd.utils", Verbosity)
 
 try:
     dataDir = lsst.utils.getPackageDir('afwdata')
+    medMIPath = os.path.join(dataDir, "data", "med.fits")
```

Comment:
I have committed that alternative, changing both tests that you commented on. I hope you like it.